exercises/resistor-color-trio/src/main/java/ResistorColorTrio.java

```diff
@@ -0,0 +1,5 @@
+class ResistorColorTrio {
+    String value(String[] colors) {
```

Comment:
```suggestion
    String value(Color... colors) {
```

This should really take the Color enum as input. It's a good setup for varargs.

Reply:
From the exercise I thought the idea behind the exercise was applying knowledge about enums. Thus predefining them might defeat the purpose.

Fairly honestly however, I have since almost fully forgotten how exercism works and/or why the entire project is set up like it is. My apologies.